scripts/appstore-fixture.mjs

```
@@ -0,0 +1,446 @@
+    .replace(/</g, '&lt;')
+    .replace(/>/g, '&gt;')
+    .replace(/"/g, '&quot;')
+    .replace(/'/g, '&#39;');
+
+const renderInlineMarkdown = (value) => {
+  const parts = [];
+  const pattern = /(`[^`]+`|\*\*[^*]+\*\*)/g;
+  let cursor = 0;
+  let match;
+
+  while ((match = pattern.exec(value)) !== null) {
```

ESLint

[error] Unexpected assignment within a 'while' statement.

(no-cond-assign)